### PR TITLE
chore: add week 5 update and project proposal

### PR DESCRIPTION
**Pull request by amintalebi**

### CODE REVIEW

File: projects/focil-geth-and-prysm-poc.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  - `AggregateInclusionLists` function: Aggregates local $IL$ and creates $ IL_{agg} $.
+  
+- **EvaluateInclusionList function:**
+
+   [![\\ $$ \\ \text{Eval}(IL_{\text{agg}}^{\text{attester}}, IL_{\text{agg}}^{\text{proposer}}, \Delta) =  \\ \begin{cases}  \\ \text{True} & \text{if } \frac{|IL_{\text{agg}}^{\text{attester}} \cap IL_{\text{agg}}^{\text{proposer}}|}{|IL_{\text{agg}}^{\text{attester}}|} \geq \Delta \\  \\ \text{False} & \text{otherwise}  \\ \end{cases} \\ ](https://latex.codecogs.com/svg.latex?%5C%5C%20%24%24%20%5C%5C%20%5Ctext%7BEval%7D(IL_%7B%5Ctext%7Bagg%7D%7D%5E%7B%5Ctext%7Battester%7D%7D%2C%20IL_%7B%5Ctext%7Bagg%7D%7D%5E%7B%5Ctext%7Bproposer%7D%7D%2C%20%5CDelta)%20%3D%20%20%5C%5C%20%5Cbegin%7Bcases%7D%20%20%5C%5C%20%5Ctext%7BTrue%7D%20%26%20%5Ctext%7Bif%20%7D%20%5Cfrac%7B%7CIL_%7B%5Ctext%7Bagg%7D%7D%5E%7B%5Ctext%7Battester%7D%7D%20%5Ccap%20IL_%7B%5Ctext%7Bagg%7D%7D%5E%7B%5Ctext%7Bproposer%7D%7D%7C%7D%7B%7CIL_%7B%5Ctext%7Bagg%7D%7D%5E%7B%5Ctext%7Battester%7D%7D%7C%7D%20%5Cgeq%20%5CDelta%20%5C%5C%20%20%5C%5C%20%5Ctext%7BFalse%7D%20%26%20%5Ctext%7Botherwise%7D%20%20%5C%5C%20%5Cend%7Bcases%7D%20%5C%5C%20)](#_)
```
Comment (taxmeifyoucan) [inline projects/focil-geth-and-prysm-poc.md:29]: This is not rendering correctly 
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions

File: projects/focil-geth-and-prysm-poc.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+- Block building rights are auctioned to specialized entities known as builders.
+- Over the past year, a few sophisticated builders have dominated block production due to economies of scale.
+
+**Censorship Resistance:**
+- Concentrated block production weakens the network’s censorship resistance properties.
+- 60% of builders filter transactions from sanctioned addresses, while 90% of validators do not engage in censorship.
+
+## Project Description
+
+A randomly selected set of validators forms an inclusion list ($IL$) committee to create and broadcast local inclusion lists. The block producer aggregates these lists into a canonical aggregate ($IL_{agg}$), which is included in the block and validated by attesters to ensure block validity.
```
Comment (taxmeifyoucan) [inline projects/focil-geth-and-prysm-poc.md:17]: A bit more context would be nice here, maybe just adding the link to the research post.

File: projects/focil-geth-and-prysm-poc.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+# Fork-Choice Enforced Inclusion Lists (FOCIL) Proof of Concept in Geth and Prysm
+
+Enhancing Decentralization and Censorship Resistance in Block Production
```
Comment (taxmeifyoucan) [inline projects/focil-geth-and-prysm-poc.md:3]: Here or in the goal of the project I would note that it's meant as prototyping for feasibility research so it's clear it's not an EIP getting to mainnet

**Review by taxmeifyoucan — COMMENTED** [review]
Thanks a lot Amin, looks great! 

@lean-apple Can you please confirm if everything sits well with you here so I can merge it?

**Comment by amintalebi** [comment]
> Thanks a lot Amin, looks great!
> 
> @lean-apple Can you please confirm if everything sits well with you here so I can merge it?

@taxmeifyoucan Lea and I are part of FOCIL working group. She's going to work on Rust implementation. I tagged her because we are woking on the same research topic. I am going to do the Golang PoC solo.

File: projects/focil-geth-and-prysm-poc.md (hunk 3 above)
Comment (amintalebi) [reply projects/focil-geth-and-prysm-poc.md:3]: sure.

File: projects/focil-geth-and-prysm-poc.md (hunk 2 above)
Comment (amintalebi) [reply projects/focil-geth-and-prysm-poc.md:17]: sure.

**Comment by taxmeifyoucan** [comment]
Got it, thanks Amin. There could be a single project proposal if you are working together but it makes sense to separate them by different implementations

**Comment by amintalebi** [comment]
@taxmeifyoucan I ran `markdown-table-formatter` for the table. That's why some more lines changed.

**Comment by taxmeifyoucan** [comment]
That's great, thanks Amin!

**Pull request merged**